Sources/MagicSDK/Modules/User/UserModule.swift

```diff
@@ -125,6 +162,12 @@ public class UserModule: BaseModule {
         updatePhoneNumber
      */
     public func updatePhoneNumber(response: @escaping Web3ResponseCompletion<String>) {
+        if #available(iOS 14.0, *) {
+            UserModule.logger.warning("updatePhoneNumber: \(BaseWarningLog.MA_Method)")
```

Comment:
Could we remove the logger？

We have a proper error message, and I think we're good here.

```
Error(code: -32603, message: "Internal error: Unsupported Magic Connect method.")
```

Reply:
Good catch!